scheduler/src/cook/mesos/api.clj

```diff
@@ -1111,8 +1112,9 @@
         ;; if the query params are coerceable, both :job and "job" keys will be
         ;; present, but in that case we only want to read the coerced version
         query-jobs (or query-jobs-coerced (mapv #(UUID/fromString %) query-jobs-uncoerced))
+        body-job (vectorize (get-in ctx [:request :body-params :job]))
```

Comment:
Thinking more, I think vectorize is particularly related to this api namespace (and to compojure-api in particular) because it was created primarily to interact with the way Compojure coerces http query parameters.   If someone passes "job=x", Compojure will serialize it to a single value, whereas "job=x&job=y" will be serialized to a Vector.